app/views/listings/form/_price.haml

```diff
@@ -33,7 +33,7 @@
                 "#fee_display",
                 "#listing_currency",
                 #{commission_from_seller},
-                #{minimum_commission},
+                #{minimum_commission.format(decimal_mark: ".", symbol: false)},
```

Comment:
This is not right for all languages. Finnish for example uses `,` as a decimal mark.